io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -3,15 +3,411 @@
   <id type="desktop-application">io.freetubeapp.FreeTube</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>FreeTube</name>
-  <summary>An Open Source YouTube app for privacy</summary>
+  <summary>An open-source YouTube app for privacy</summary>
+  <summary xml:lang="ar">تطبيق يوتيوب مفتوح المصدر للخصوصية</summary>
+  <summary xml:lang="az">Gizlilik üçün açıq mənbəli YouTube proqramı</summary>
+  <summary xml:lang="be">Праграма YouTube з адкрытым зыходным кодам для забеспячэння прыватнасці</summary>
+  <summary xml:lang="bg">Приложение за YouTube с отворен код за поверителност</summary>
+  <summary xml:lang="bn">গোপনীয়তার জন্য একটি ওপেন-সোর্স YouTube অ্যাপ</summary>
+  <summary xml:lang="ca">Una aplicació de YouTube de codi obert per a la privadesa</summary>
+  <summary xml:lang="cs">Open-source aplikace YouTube pro ochranu soukromí</summary>
+  <summary xml:lang="da">En open source YouTube-app til beskyttelse af personlige oplysninger</summary>
+  <summary xml:lang="de">Eine Open-Source-YouTube-App für den Datenschutz</summary>
+  <summary xml:lang="el">Μια εφαρμογή YouTube ανοιχτού κώδικα για προστασία απορρήτου</summary>
+  <summary xml:lang="en_AU">An open-source YouTube app for privacy</summary>
```

Comment:
I dont see this anywhere on https://flathub.org/languages. Remove all translations related to this lang

Reply:
If only the en-AU English locale installed (that is en-US and en-GB are not installed), then the en-US shows. For your app en-AU/en-GB are the same as en-US; however, there are apps where it is different, eg MMEX.